ui/src/message_popup/playtest/smithA_scenario.jsx

```diff
@@ -0,0 +1,401 @@
+/* @flow weak */
+import React from 'react';
+
+/*
+This file defines the content for the scenario around bubble sort.
+*/
+
+export type QuestionT = {
+  type:string, // Used as a label
+  text:string,
+  open:?bool, // Ask for open-ended user response
+  choices:?bool // Forced-choice response
+};
+
+
+function slidesFor(cohortKey) {
+  const slides:[QuestionT] = [];
+
+  slides.push({ type: 'Overview', el:
+    <div>
+    <div>1. Set Context</div>
+    <div>Imagine yourself as a teacher situated in the context of the particular school, classroom and subject.</div>
+    <br />
+    <div>2. Background</div>
+    <div>A little bit of background information on the lesson being taught.</div>
+    <br />
+    <div>3. Try it!</div>
+    <div>When you're ready, you'll go through a set of short scenes that simulate moments in the classroom. Note what you observe.</div>
+    <br />
+    <div>4. PAUSE!</div>
+    <div>Pause for a brief group discussion before resuming.</div>
+    <br />
+    <div>5. Lenses</div>
+    <div>Try another set of scenes, this time from a new perspective.</div>
+  </div>
+  });
+
+
+// Context
+  slides.push({type: 'Context', text: 
+      `Imagine you're a teacher at a suburban high school. Your friend and colleague, Mr. Smith, asks you to sit in and observe his CS class for a day.
+
+The class, taught by Mr. Smith, has 6 students. The demographics of the class are as follows:
+
+      5 Male Students
+      1 Female Student
+      4 White Students (3 Male, 1 Female)
+      2 African American Students (2 Male)`
+  });
+
+
+
+  // -----------------------------------------------------
+  // BACKGROUND
+  // -----------------------------------------------------
+
+  slides.push({type: 'Background', text: 
+      `Mr. Smith will be teaching a class on encoding and sending formatted text. 
+
+In doing so, his students will be working with the standard internet encoding protocol, ASCII. The class will be using pair programming to complete a project developing their own protocol for sending formatted (i.e bold, italicized, etc.) text. 
+
+      `
+    });
+
+  slides.push({type: 'Background', el: 
+<div>
+  <div>Useful Knowledge:</div>
+  <br />
```

Comment:
This scenario uses a lot more `el` scenes, where you're passing React elements.  We can probably figure out a better way to structure that, since when we're writing rendering code we'd usually use styles or CSS to control the spacing rather than adding `<br />` tags like the original example you copied from does (my old mistake).  Let's leave that for now and we can revisit afterward.